backend/src/nodes/nodes/pytorch/convert_to_onnx.py

```diff
@@ -19,17 +20,37 @@
 class ConvertTorchToONNXNode(NodeBase):
     def __init__(self):
         super().__init__()
-        self.description = """Convert a PyTorch model to ONNX."""
-        self.inputs = [ModelInput("PyTorch Model")]
-        self.outputs = [OnnxModelOutput(label="ONNX Model")]
+        self.description = """Convert a PyTorch model to ONNX.
+            Note: fp16 conversion will only work if PyTorch fp16 mode is turned on."""
+        self.inputs = [
+            ModelInput("PyTorch Model"),
+            OnnxFpDropdown(),
+        ]
+        self.outputs = [
+            OnnxModelOutput(label="ONNX Model"),
+            TextOutput(
+                "FP Mode",
+                """match Input1 {
+                    FpMode::fp32 => "fp32",
+                    FpMode::fp16 => "fp16",
+                }""",
+            ),
+        ]
 
         self.category = PyTorchCategory
         self.name = "Convert To ONNX"
         self.icon = "ONNX"
         self.sub = "Utility"
 
-    def run(self, model: PyTorchModel) -> OnnxModel:
+    def run(
+        self, model: PyTorchModel, is_fp16: int = 0, final: bool = True
```

Comment:
> `final: bool = True`

No. The parameters of the `run` method as supposed to mirror the inputs of the node (except in the case of some iterator helper nodes). 

I also don't get why this parameter is even necessary. Couldn't you just us the existing `is_fp16`?

Reply:
I don't think it is now. I put it there before I decided to add the fp32/fp16 dropdown, so I needed a way to indicate that it should not just follow fp16 mode in settings.